skimage/util/tests/test_invert.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from skimage import dtype_limits
+from skimage.util.invert import invert
```

Comment:
I think it is better to rename `invert.py` to `_invert.py` and use `from skimage.util import invert` here.